tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/bala/constant/MapConstantEqualityInDifferentBalaTest.java

```diff
@@ -35,6 +35,7 @@ public class MapConstantEqualityInDifferentBalaTest {
 
     @BeforeClass
     public void setup() {
+        BCompileUtil.compileAndCacheBala("test-src/bala/test_projects/test_project_utils");
```

Comment:
Given that what we are introducing is a simple test, I would extract that test and packages out separately, and avoid these unnecessary compilations for other tests.

Reply:
@chiranSachintha, please send a separate PR fixing this, will merge this for now since it is required for U9.

Reply:
Updated the test case